src/Compilers/VisualBasic/Portable/Symbols/MethodSymbol.vb

```diff
@@ -141,6 +141,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         End Function
 
         ''' <summary>
+        ''' Build and add synthesized return type attributes for this method symbol.
+        ''' </summary>
+        Friend Overridable Sub AddSynthesizedReturnTypeAttributes(ByRef attributes As ArrayBuilder(Of SynthesizedAttributeData))
```

Comment:
we do the same thing in C#, but now I'm confused about it as well...

---
In reply to: [79711243](https://github.com/dotnet/roslyn/pull/13923#discussion_r79711243) [](ancestors = 79711243)